libnavigation-core/src/test/java/com/mapbox/navigation/core/internal/formatter/MapboxDistanceFormatterTest.kt

```diff
@@ -258,7 +258,7 @@ class MapboxDistanceFormatterTest {
                 .build()
         ).formatDistance(-0.1)
 
-        assertEquals("0 ft", result.toString())
+        assertEquals("$INCREMENT_FIFTY ft", result.toString())
```

Comment:
Why are we defaulting to `INCREMENT_FIFTY` if the distance is below zero? Should it be `5 ft` instead? Also see comment above re: negative distances.

Reply:
The minimum value returned will be the rounding increment.